support/Pipelines/Bbh/Inspiral.py

```diff
@@ -147,8 +147,8 @@ def inspiral_parameters(
             / (id_input_file["Observers"]["VolumeFileName"] + "*.h5")
         ),
         # Domain geometry
-        "ExcisionRadiusA": id_domain_creator["ObjectA"]["InnerRadius"],
-        "ExcisionRadiusB": id_domain_creator["ObjectB"]["InnerRadius"],
+        "ExcisionRadiusA": id_domain_creator["ObjectA"]["InnerRadius"] * 1.0385,
+        "ExcisionRadiusB": id_domain_creator["ObjectB"]["InnerRadius"] * 1.0385,
```

Comment:
Just need to add the factor for the spec ID from #6316

Reply:
squashed that in :)

Reply:
This factor comes from dividing the excision radius used in the paper by what develop outputs for the excision radius when using the same initial data. I guess I could increase the initial data, I didn't think about that, that way there would only be one number floating around changing the excision radius.

Reply:
I think this is because the ID excision in the last iteration of the control loop is not the same shape as the final horizon found after the last iteration. But then the inspiral excision is set based off this final horizon mass/spin. Therefore it's possible they don't line up properly, so making the inspiral excision slightly bigger accounts for this.

Reply:
Hm can we just start from the ID excision, using the same initial shape for the excision as the ID? Not just the same inner radius, but set the initial shape to the same Kerr-Schild parameters as well (instead of using the AH shape). This would avoid empirically found and potentially fragile factors like this one.

Reply:
We could but then this would cause shape/size control to take longer (or potentially many more iterations) to settle to the correct shape/size since we're controlling to the horizon.

I'd like to get what we did for the paper onto develop so we at least have it, and then we can decide what's best to do after. There's quite a lot more automation that has to happen and decisions to be made about what's best/robust. Even if we use the ID excision shape/size, we'll likely change that anyways.